File: js/factorial-tail.js
function zeroes (base, number) {
  var factors = {}, i = 1;
  while(++i <= base) 
    while(base%i == 0) {
      base /= i; 
      factors[i] = (factors[i]||0) + 1;
    }

  return Math.min(...Object.keys(factors).map(factor => {
    var count = 0, i = 1;
    while((i *= factor) <= number) count += Math.floor(number/i);
    return count/factors[factor];
  }));
}

File: python/FactorialTail.py
def zeroes(base, number):
  factors = factor(base)
  result = []
  for x in factors.keys():
    count, i = 0, x
    while i <= number:
      count += number / i
      i *= x
    result.append(count / factors[x])
  return min(result)

def factor(base):
  d = 2
  dic = {}
  while d <= base:
    c = 0
    while base % d == 0:
      c += 1
      base /= d
    if c != 0:
      dic[d] = c
    d += 1
  return dic

print(zeroes(60, 7753))